test_runner/regress/test_lfc_prewarm.py

```diff
@@ -19,11 +24,23 @@ def check_pinned_entries(cur):
     assert n_pinned == 0
 
 
+def prom_parse(client: EndpointHttpClient) -> dict[str, float]:
+    return {
+        sample.name: sample.value
+        for family in prom_parse_impl(client.metrics())
+        for sample in family.samples
+        if sample.name in (PREWARM_LABEL, OFFLOAD_LABEL)
+    }
+
+
 @pytest.mark.skipif(not USE_LFC, reason="LFC is disabled, skipping")
-def test_lfc_prewarm(neon_simple_env: NeonEnv):
+@pytest.mark.parametrize("with_compute_ctl", [False, True], ids=["pg-only", "compute-ctl"])
+def test_lfc_prewarm(neon_simple_env: NeonEnv, with_compute_ctl: bool):
+    """
+    with_compute_ctl: Test compute ctl's methods instead of querying Postgres directly
```

Comment:
I typically think that booleans can be overloaded. For instance, you have to write this docstring to explain what `with_compute_ctl` means. I like to use enums in such cases.

```
from enum import Enum

class LfcQueryMethod(Enum)
    COMPUTE_CTL
    POSTGRES
```

Makes it a little more obvious. But I will leave it up to you, merely a suggestion on improving readability